app.config.js

```diff
@@ -0,0 +1,21 @@
+import * as dotenv from "dotenv";
+import * as fs from "fs";
+
+const ENV = process.env.APP_ENV || "development";
+const envFile = `.env.${ENV}`;
```

Comment:
- i do not think we need multiple .env files for staging, dev (or) prod as in the ci we will generate the env file and set the appropriate `APP_ENV` depending on if the ci is running in dev / staging / prod